cmake/FindRust.cmake

```diff
@@ -221,6 +226,24 @@ directory = \".cargo/vendor\"
     endif()
 endfunction()
 
+# Export the project name (including upstream project) and maintainer mode
+# status for use by build scripts
+function(GetCargoExtraEnv result)
```

Comment:
Naming convention for CMake functions seems to be snake_case. Can we switch this from CamelCase?

Reply:
Absolutely.  I think I picked this style up from another file I was using as reference.